.github/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
```

Comment:
- [ ] Please add linters properly, this particular linter should be as `.github/workflows` and then .linters.yml
- [ ] Also add `.hintrc`

_**If you need a refresher for linters you can visit [LINTERS](https://github.com/microverseinc/linters-config/tree/master/html-css)**_